tests/ps_test.go

```diff
@@ -61,7 +61,7 @@ var _ = Describe("Processes", func() {
 			})
 		})
 
-		DescribeTable("can scale up and down",
+		PDescribeTable("can scale up and down",
```

Comment:
If this is marked pending, will the associated regular/non-pending `Entry` lines below run?